src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -207,4 +208,28 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addTwigFormatOptions(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('date')
```

Comment:
should add info that this section configured the defaults behavior of the `date` filter